### DISCUSSION TITLE
Monday Meeting (Finals week)

### DISCUSSION THREAD

**Discussion by Emend026** [anhourandaquarter/carmina#32] [General]
Hi guys. Please share your availability for Monday. I'll send out a zoom link once I have everyone's schedule. 

mon: free after 12pm :]

**Comment by Emend026** [comment]
Oh also can yall send me a picture of you guys for our bio slide? *please* :]

**Comment by infinite0x20** [comment]
I'm free 3pm and after! I can email you my picture : )

**Comment by hhchan94** [comment]
Any time  except  1 to 3 works

**Comment by infinite0x20** [comment]
OK, can we go for 4:30pm?

**Comment by Emend026** [comment]
@thatswhatsimonsaid @hhchan94 don't forget to send me your pics please :]

**Comment by infinite0x20** [comment]
Did we create a zoom link? If not, I can provide one

EDIT: Sorry, misplaced it in my inbox 😔

**Comment by Emend026** [comment]
yes it's on the calendar. here too: Join Zoom Meeting
https://washington.zoom.us/j/9539034249
<https://www.google.com/url?q=https://washington.zoom.us/j/9539034249&sa=D&source=calendar&ust=1734149530650539&usg=AOvVaw1l7PECu0PUlwxTEs37P8t->

On Mon, Dec 9, 2024 at 4:31 PM Suh Young Choi ***@***.***>
wrote:

> Did we create a zoom link? If not, I can provide one
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/anhourandaquarter/carmina/discussions/32*discussioncomment-11514407__;Iw!!K-Hz7m0Vt54!jcq6rEukmkS7oPtSXM9wVuuh8nL-ssL6qceslODUNreNTxEEB2Zoi1jTyn88VjMdylbCGcx8T10hm7JwfKP-3DmZuQ$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AWMXGZ6A3C4LOPI5VJ76KP32EYY5ZAVCNFSM6AAAAABTFJRKAGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNJRGQ2DANY__;!!K-Hz7m0Vt54!jcq6rEukmkS7oPtSXM9wVuuh8nL-ssL6qceslODUNreNTxEEB2Zoi1jTyn88VjMdylbCGcx8T10hm7JwfKMDDVqc8Q$>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Elizabeth Nova
PhD Student
Department of Sociology
University of Washington

**Comment by Emend026** [comment]
@hhchan94 heyy will you be able to make it?

**Reply by Emend026** [reply]
meeting ended. I shared access to the presentation slides so feel free to take a look. Also @hhchan94 could you send me a picture to add for our bio slide please? :) tysm!

**Reply by hhchan94** [reply]
I am sorry for missing the meeting earlier. I was preparing for my final tomorrow and completely lost track of time. Since the meeting has already ended, I have a few questions for @anhourandaquarter regarding scansion. Would you have a minute to meet tonight? Thanks!

**Reply by hhchan94** [reply]
Also @Emend026  I just add my photo to the slide. Thank you!
